integration_tests/structs_27.py

```diff
@@ -7,6 +7,14 @@ class StringIO:
     _0cursor : i32 = 10
     _len     : i32 = 1
 
+@dataclass
+class StringIONew:
+    _buf     : str
+    _0cursor : i32 = i32(142)
+    _len     : i32 = i32(2439)
```

Comment:
This PR assigns this to the "value" member of Variable. I am not sure if our code assumes that this is a constant, so the backend doesn't even visit this variable and uses the value instead. If so, then this won't work, since any modifications to `_len` will be lost.

Reply:
Let's add a test for modifying `_len`.

Reply:
Here is documentation for Variable:
```
`symbolic_value` the optional symbolic expression to initialize the variable
(e.g. `2+3+4+x`), this value must be compile time, but it is not necessarily a
constant (e.g., can contain binary operations, other variables, etc.)

`value` the optional constant expression holding the compile time value
(e.g. `5`, or `5.5`), it is a compile time constant.
```
So `value` and `symbolic_value` must be compile time constants. So let's require that for now as well in LPython.

A separate question is if they are set, if the `Variable` itself has to be a constant or not. I would say if it has to be a constant, then it should have a type Const. Otherwise it can change, which means `value` is the initial value, but the runtime value can change.

Reply:
Yeah, so I tested, and if the variable is having a value then it doesn't get changed after the assignment, so basically, it should only be used with the const variables.

Now, this problem gets trickier as Structs don't have a body, and, so `AnnAssign` which is an Assignment to the struct variables isn't possible with the current design.

Reply:
One thing that we may do is add a field in the Struct to contain initializers for all it's variable members like the following:
```diff
     | StructType(symbol_table symtab, identifier name, identifier* dependencies,
         identifier* members, abi abi, access access, bool is_packed, bool is_abstract,
-        expr? alignment, symbol? parent)
+        expr* initializers, expr? alignment, symbol? parent)
```